AlphaWallet/Settings/Models/AddCustomChain.swift

```diff
@@ -238,3 +243,27 @@ extension AddCustomChain.functional {
         }
     }
 }
+
+class EditCustomChain: AddCustomChain {
```

Comment:
Instead of subclassing `AddCustomChain`, can you add an enum property to `AddCustomChain` instead? Less fragile.

So it becomes:

```
enum CustomChainOperationType {
    case .add
    case .edit //includes `originalCustomRPC` as associated value if necessary
}

enum AddCustomChain {
    private let operationType: CustomChainOperationType

    //existing code
}
```

We out to rename `AddCustomChain`, but let's just leave it out first, to reduce the diffs.

Do you think that's better?